addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -782,14 +816,17 @@ protected void onFetchData() {
                     channelDaemonFrame = false;
                 }
             }
-        } finally {
+        } finally
+
```

Comment:
Sorry I misunderstood your first comment "Same for all other occurences below", as if it was the following comments. Reading it more carefully gives fully sense, off course the above change should be reverted :-)